protected/controllers/PatientController.php

```diff
@@ -1497,6 +1497,7 @@ public function actionCreate()
         $practicecontact = new Contact('manage_practice');
         $practiceaddress = new Address('manage_practice');
         $practice = new Practice('manage_practice');
+        $patientuserreferral = null;
```

Comment:
Style: variables names should either use $camelCase or $under_scores between words